eng/common/scripts/Prepare-Release.ps1

```diff
@@ -127,6 +128,17 @@ if ($null -eq $newVersionParsed)
   exit 1
 }
 
+# Check API status if version is GA
+if (-not $newVersionParsed.IsPrerelease)
+{
+  $isApproved = Check-ApiReviewStatus -PackageName $packageProperties.Name -Language $LanguageDisplayName
+  if (-not $isApproved)
+  {
+    Write-Error "API review is not approved in APIView system."
```

Comment:
It would be good to have an aka link with more information for people to be able to fix this.

Reply:
Nevermind I see you included those in the function you are calling to check.